backend/apps/owasp/apps.py

```
@@ -7,3 +7,6 @@ class OwaspConfig(AppConfig):
     """Owasp app config."""
 
     name = "apps.owasp"
+
+    def ready(self):
```

Ruff (0.14.14)

[warning] Missing docstring in public method

(D102)